doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -1,492 +0,0 @@
-    
-    arr.meta
-
-
-- markdown: |
-    <div class="alert alert-warning">
-    
-    **Warning:**  
-      <ul>
-        <li>Currently, only the HDF (.h5) file format supports saving and loading array metadata.</li>
-        <li>Metadata is not kept when actions or methods are applied on an array
-            except for operations modifying the object in-place, such as `population[age < 10] = 0`,
-            and when the method `copy()` is called. Do not add metadata to an array if you know
-            you will apply actions or methods on it before dumping it.</li>
-      </ul>
-    
-    </div>
-
-
-- markdown: |
-    ### Array creation functions
-    
-    Arrays can also be generated in an easier way through [creation functions](../api.rst#array-creation-functions):
-    
-    -  `ndtest` : creates a test array with increasing numbers as data
-    -  `empty` : creates an array but leaves its allocated memory
-       unchanged (i.e., it contains "garbage". Be careful !)
-    -  `zeros`: fills an array with 0
-    -  `ones` : fills an array with 1
-    -  `full` : fills an array with a given value
-    -  `sequence` : creates an array from an axis by iteratively applying a function to a given initial value.
-    
-    Except for ndtest, a list of axes must be provided.
-    Axes can be passed in different ways:
-    
-    -  as Axis objects
-    -  as integers defining the lengths of auto-generated wildcard axes
-    -  as a string : 'gender=M,F;time=2007,2008,2009' (name is optional)
-    -  as pairs (name, labels)
-    
-    Optionally, the type of data stored by the array can be specified using argument dtype.
-
-
-- code: |
-    # start defines the starting value of data
-    ndtest((3, 3), start=-1)
-
-
-- code: |
-    # start defines the starting value of data
-    # label_start defines the starting index of labels
-    ndtest((3, 3), start=-1, label_start=2)
-
-
-- code: |
-    # empty generates uninitialised array with correct axes
-    # (much faster but use with care!).
-    # This not really random either, it just reuses a portion
-    # of memory that is available, with whatever content is there.
-    # Use it only if performance matters and make sure all data
-    # will be overridden.
-    empty([age, gender, time])
-
-
-- code: |
-    zeros([age, gender, time])
-
-
-- code: |
-    # dtype=int forces to store int data instead of default float
-    ones([age, gender, time], dtype=int)
-
-
-- code: |
-    full([age, gender, time], fill_value=1.23)
-
-
-- markdown: |
-    All the above functions exist in *(func)_like* variants which take axes from another array
-
-
-- code: |
-    ones_like(arr)
-
-
-- markdown: |
-    Create an array using the special ``sequence`` function (see link to documention of ``sequence`` in API reference for more examples):
-
-
-- code: |
-    # With initial=1.0 and inc=0.5, we generate the sequence 1.0, 1.5, 2.0, 2.5, 3.0, ...
-    sequence(age, initial=1.0, inc=0.5)
```

Comment:
I **wonder** if this example is not better than the one in Getting started.

Reply:
There is one. Just below the title "## Grouping arrays in a Session".

Reply:
I guess this comment was meant as an answer to the previous thread....

Reply:
> There is one. Just below the title "## Grouping arrays in a Session".

It's not very visible IMO. Case in point, I did not see it. I think that an explicit mention like what we had in the "presenting objects" before would help.

> Finally, see the [Session](../api.rst#session) section of the API Reference to explore all methods of Session objects.